.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# Lines starting with '#' are comments.
+
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '\*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+
+# This should make it easy to add new rules without breaking existing ones.
+
+- @grafana/aws-plugins
```

Comment:
I haven't seen the `-` syntax in our files before, does it work the same way as the `*` we use in other repos? https://github.com/grafana/athena-datasource/blob/main/.github/CODEOWNERS

Reply:
good catch. one of my vscode extensions auto-formats this as a markdown file on save. didn't notice it was doing this cause i just copied the codeowners file from another repo.